flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -374,6 +374,15 @@ public class ExecutionConfigOptions {
     // ------------------------------------------------------------------------
     //  Async Lookup Options
     // ------------------------------------------------------------------------
+    @Documentation.TableOption(execMode = Documentation.ExecMode.BATCH_STREAMING)
+    public static final ConfigOption<Boolean> TABLE_EXEC_ASYNC_LOOKUP_KEY_ORDERED =
+            key("table.exec.async-lookup.key-ordered-enabled")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "When true async lookup joins would follow the upsert key order in cdc stream (if no upsert key, "
```

Comment:
nits:
I was not sure what `true async lookup joins ` were. I suggest 
`When true, async lookup joins follow the upsert key order in the CDC stream.`
I suggest removing the brackets and having a new sentence.
`If there is no defined upsert key, then the total record is considered as the upsert key.`

nit: This feature does not works for insert-only. -> setting this for insert-only has no effect.

Could we say why this has no effect for insert only? is it not possible to order the inserts by key in this case?

Reply:
Am I correct in saying , by default the order with be time order. With this option the keys will be ordered alphabetically if a string or numerically if a number. We should mention this in the description. Is it possible to have other types as keys?